src/script/lua_api/l_env.cpp

```diff
@@ -611,7 +611,8 @@ int ModApiEnvMod::l_find_node_near(lua_State *L)
 		ndef->getIds(lua_tostring(L, 3), filter);
 	}
 
-	for(int d=1; d<=radius; d++){
+	int start_at = !lua_toboolean(L, 4);
```

Comment:
Yes kaeza's suggestion here is better. Also a better variable name would be good, 'min_radius'?